conformance/tests/classes_classvar.py

```diff
@@ -0,0 +1,156 @@
+"""
+Tests the typing.ClassVar special form.
+"""
+
+# Specification: https://typing.readthedocs.io/en/latest/spec/class-compat.html#classvar
+
+from typing import (
+    Annotated,
+    Any,
+    Callable,
+    ClassVar,
+    Generic,
+    ParamSpec,
+    Protocol,
+    TypeAlias,
+    TypeVar,
+    TypeVarTuple,
+    assert_type,
+)
+from typing import ClassVar as CV
+
+
+var1 = 3
+
+P = ParamSpec("P")
+T = TypeVar("T")
+Ts = TypeVarTuple("Ts")
+
+# This is currently commented out because it causes mypy to crash.
+# class ClassA(Generic[T, P, Unpack[Ts]]):
+
+
+class ClassA(Generic[T, P]):
+    # > ClassVar accepts only a single argument that should be a valid type
+
+    bad1: ClassVar[int, str]  # Type error: too many arguments
+    bad2: CV[3]  # Type error: invalid type
+    bad3: CV[var]  # Type error: invalid type
+
+    # > Note that a ClassVar parameter cannot include any type variables,
+    # > regardless of the level of nesting.
+
+    bad4: ClassVar[T]  # Type error: cannot use TypeVar
+    bad5: ClassVar[list[T]]  # Type error: cannot use TypeVar
+    bad6: ClassVar[Callable[P, Any]]  # Type error: cannot use ParamSpec
+
+    # This is currently commented out because it causes mypy to crash.
+    # bad7: ClassVar[tuple[*Ts]]  # Type error: cannot use TypeVarTuple
+
+    bad8: ClassVar[list[str]] = {}  # Type error: type violation in initialization
+
+    bad9: Final[ClassVar[int]] = 3  # Type error: ClassVar cannot be nested with Final
+    bad10: list[ClassVar[int]] = []  # Type error: ClassVar cannot be nested
+
+    good1: CV[int] = 1
+    good2: ClassVar[list[str]] = []
+    good3: ClassVar[Any] = 1
+    good4: ClassVar = 3.1
+    good5: Annotated[ClassVar[list[int]], ""] = []
```

Comment:
Should we also check ClassVar[Annotated ? Not sure the spec covers this

Reply:
This is covered in the `qualifiers_annotated.py` test.